source/gui/__init__.py

```diff
@@ -596,6 +596,8 @@ def initialize():
 	if mainFrame:
 		raise RuntimeError("GUI already initialized")
 	mainFrame = MainFrame()
+	wxLang = core.getWxLang(languageHandler.getLanguage())
+	mainFrame.SetLayoutDirection(wxLang.LayoutDirection)

```

Comment:
What if `wxLang` is returned `None` from `getWxLang`

Reply:
I've handled this by 

- leaving it with default behaviour 
- moved a warning into `getWxLang` when it cannot be found 
- renaming the function to `getWxLangOrNone`